tests/MongoDB.Driver.Tests/Linq/Linq3ImplementationTests/Serializers/KnownSerializers/KnownSerializerFinderTests.cs

```diff
@@ -0,0 +1,222 @@
+/* Copyright 2010-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Linq.Expressions;
+using FluentAssertions;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Driver.Linq.Linq3Implementation.Serializers.KnownSerializers;
+using Xunit;
+
+namespace MongoDB.Driver.Tests.Linq.Linq3ImplementationTests.Serializers.KnownSerializers
+{
+    public class KnownSerializerFinderTests
+    {
+        #region static
+        private static IBsonDocumentSerializer GetCollectionSerializer()
```

Comment:
I'm not a fan of static methods just because "they can be" static...

My bar for a static method is higher:

1. A method that you call *without* having an instance of the class
2. A method called from another static method
3. A method called from a constructor

That's about it... everything else is an instance method, specially if called from another instance method. 

Having said that... I'm OK with LGTM'ing as-is if that's what you want.

Reply:
My criteria for statics is “If a method doesn’t require an instance, make it a static. If that looks wrong, then there may be something wrong with your design.” A lot of static methods usually indicates that you’re not assigning responsibilities correctly and your code is more procedural than OO.

I'm OK with making this helper an instance method and moving it to the bottom, which is what I've done.